src/interactive_list.js

```diff
@@ -0,0 +1,26 @@
+import { setLocalStorage, addTask } from './addremove.js';
+
+let Task = JSON.parse(localStorage.getItem('Task')) || [];
+
+const editCheckBox = (ID, event) => {
+  Task.forEach((element) => {
+    if (element.index === ID) {
+      element.completed = event;
+    }
+    setLocalStorage(Task);
+  });
+};
```

Comment:
- [ ] Kindly check on your `editCheckBox` function, I cant seem to save a completed task in my local storage